### DISCUSSION TITLE
CapRover one-click app for Kamai

### DISCUSSION THREAD

**Discussion by deviousjack** [kimai/kimai#4100] [General]
Incase anyone is interested and uses CapRover, I have a one-click app for trying Kamai with the docker image. CapRover seems good for "we" less technical folk to use Docker with ease.

CapRover does have Kamai in it's one-click-apps repo but hasn't been updated in an eternity and doesn't work.

https://github.com/CurbSoftwareInc/caprover-curb-click-apps

Also wanted to let the  Kamai team know to make sure they are cool with it - I referenced throughout my repo.

**Comment by kevinpapst** [comment]
Thanks so much for your support 👍 

But: wouldn't it be better to update the upstream recipe at https://github.com/caprover/one-click-apps/blob/master/public/v4/apps/kimai2.yml with your code?

**Reply by kevinpapst** [reply]
Ping @lcx just in case you are still using Kimai, maybe you want to join the discussion?

**Reply by deviousjack** [reply]
Yeah, you're right, they have their own specific rules for their official repo I think, like they don't allow using "latest" as a placeholder variable, which I like to I don't need to keep going to DockerHub to check everytime, keeping for example "apache-latest" for Kimai because things could change I suppose.  But I can just make my own repo, on bottom of caprover one-click-apps repo it gives easy instructions to lint it and publish it on GH pages so best of both worlds.  Cool, thanks for the reply.

**Comment by lcx** [comment]
sorry I didn't update the one click setup in caprover. I am still using Kimai but so far haven't upgraded and am still on 1.15 (sorry) so I guess it might not work with the new version. 
For now I'm planing to move my setup to simple docker-compose files + Portainer instead of caprover. Caprover is great for testing and setting up new things, but doing upgrades (like Kimai 1.5 to current version) can get tricky on caprover and this is why I didn't have the time to fix it. Feel free to update anything I have added to caprover to work with the new version of Kimai.
